hooks/terraform_checkov.sh

```diff
@@ -14,6 +14,10 @@ function main {
   common::initialize "$SCRIPT_DIR"
   common::parse_cmdline "$@"
   common::parse_and_export_env_vars
+  # Support for setting PATH to repo root.
+  # shellcheck disable=SC2178 # It's the simplest syntax for that case
+  ARGS=${ARGS[*]/__GIT_WORKING_DIR__/$(pwd)\/}
+  # shellcheck disable=SC2128 # It's the simplest syntax for that case
   # shellcheck disable=SC2153 # False positive
   common::per_dir_hook "${ARGS[*]}" "$HOOK_ID" "${FILES[@]}"
```

Comment:
Let's maintain one style across the codebase. Also, no need to expand ARGS to an array
```suggestion
  # shellcheck disable=SC2128 # It's the simplest syntax for that case
  common::per_dir_hook "$ARGS" "$HOOK_ID" "${FILES[@]}"
```

https://github.com/antonbabenko/pre-commit-terraform/blob/598bf2c7edd92f559b25655b899c4c41fb4d735b/hooks/terraform_tfsec.sh#L16-L20